tests/data_store/test_aws.py

```diff
@@ -226,5 +226,76 @@ def test_run_flow(self, emr):
         assert status_code == 200
         job_flow_id = response.get('JobFlowId')
         assert job_flow_id
-        cluster_info = emr._emr.describe_cluster(ClusterId=job_flow_id)
-        assert cluster_info.get("Cluster", {}).get("Name") == "HPF_training"
+        job_status = emr.get_status(job_flow_id)
+        assert job_status.get('State') == 'WAITING'
+
+    def test_terminate_job(self, emr):
```

Comment:
Missing doc-style defs

Reply:
Thanks, @miteshvp. Actually, I am not adding doc-styles for any of the tests.